tests/cases/fourslash/refactorConvertToOptionalChainExpression_ConditionalForAny.ts

```diff
@@ -0,0 +1,16 @@
+/// <reference path='fourslash.ts' />
+
+// @strict: true
+
+////interface Foo {
+////    bar?:{
+////        baz?: any;
+////    }
+////}
+////declare let foo: Foo;
+/////*a*/foo.bar ? foo.bar.baz : "whenFalse";/*b*/
+
+// It is reasonable to offer a refactor when baz is of type any since using any in strict mode

```

Comment:
```suggestion
// It is reasonable to offer a refactor when baz is of type any since implicit any in strict mode
```

(and explicit anys are known suspect places, although I don't know if that needs to be stated)